Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -0,0 +1,318 @@
+        }
+
+    }
+
+    public struct HelpPath {
+
+        public let id: String
+        public let title: String
+        public let type: PathType
+        public let detail: PathDetail?
+
+        public init(id: String,
+                    title: String,
+                    type: PathType,
+                    detail: PathDetail?) {
+            self.id = id
+            self.title = title
+            self.type = type
+            self.detail = detail
+        }
+
+        public enum PathDetail {
+
+            case promotionalOffer(PromotionalOffer)
+            case feedbackSurvey(FeedbackSurvey)
+
+        }
+
+        public enum PathType: String {
+
+            case missingPurchase = "MISSING_PURCHASE"
+            case refundRequest = "REFUND_REQUEST"
+            case changePlans = "CHANGE_PLANS"
+            case cancel = "CANCEL"
+            case unknown
+
+            init(from rawValue: String) {
+                switch rawValue {
+                case "MISSING_PURCHASE":
+                    self = .missingPurchase
+                case "REFUND_REQUEST":
+                    self = .refundRequest
+                case "CHANGE_PLANS":
+                    self = .changePlans
+                case "CANCEL":
+                    self = .cancel
+                default:
+                    self = .unknown
+                }
+            }
+
+        }
+
+        public struct PromotionalOffer {
+
+            public let iosOfferId: String
+
+            public init(iosOfferId: String) {
+                self.iosOfferId = iosOfferId
+            }
+
+        }
+
+        public struct FeedbackSurvey {
+
+            public let title: String
+            public let options: [Option]
+
+            public init(title: String, options: [Option]) {
+                self.title = title
+                self.options = options
+            }
+
+            public struct Option {
+
+                public let id: String
+                public let title: String
+
+                public init(id: String, title: String) {
+                    self.id = id
+                    self.title = title
+                }
+
+            }
+
+        }
+
+    }
+
+    public struct Appearance {
+
+        let mode: AppearanceMode
+        let light: AppearanceCustomColors
+        let dark: AppearanceCustomColors
+
+        public init(mode: AppearanceMode, light: AppearanceCustomColors, dark: AppearanceCustomColors) {
+            self.mode = mode
+            self.light = light
+            self.dark = dark
+        }
+
+        public struct AppearanceCustomColors {
+
+            let accentColor: String
+            let backgroundColor: String
+            let textColor: String
+
+            public init(accentColor: String, backgroundColor: String, textColor: String) {
+                self.accentColor = accentColor
+                self.backgroundColor = backgroundColor
+                self.textColor = textColor
+            }
```

Comment:
There are changes to this struct in #3983 which makes it use `PaywallColor`, so in this PR it's just strings and a mode enum for simplification.